src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -26,27 +28,31 @@ def _main(settings, optlist, args):
             xunitrunner
                 [--config config.json]
                 [--setting name=value]
-                --script
-                [--args arg1 arg2...]
+                --script=path
+                [--standalone]
```

Comment:
Ok let me ask in a different way.  How is this different than the hack of parsing out the --script part of the command? For both implementations you are adding knowledge of a very specific script to the repro tool which ideally we would like to avoid.

---
In reply to: [155029516](https://github.com/dotnet/buildtools/pull/1825#discussion_r155029516) [](ancestors = 155029516)